<?xml version="1.0" encoding="utf-8"?>
<report
	progress="">
	<activity
		result="Failed"
		duration="53.17s"
		durationms="53173"
		type="root"
		rid="f195a80a6bda3be"
		totalerrorcount="1"
		totalwarningcount="0"
		totalsuccesstestcasecount="0"
		totalfailedtestcasecount="0"
		totalblockedtestcasecount="0"
		childsuccesscount="0"
		childfailedcount="0"
		childblockedcount="0"
		totalsuccesscount="0"
		totalfailedcount="0"
		totalblockedcount="0"
		totalmaintenancemodecount="0"
		user="THI1901453"
		rxversion="9.1.0.0"
		host="THSLT10102"
		osversion="Windows 10 64bit"
		runtimeversion="4.0.30319.42000"
		procarch="32bit"
		oslanguage="en-US"
		dtnlanguage="en-US"
		screenresolution="1366x768"
		timestamp="6/27/2019 4:06:12 PM"
		timestampiso="2019-06-27T10:36:12"
		endtime="6/27/2019 4:07:05 PM"
		timeoutfactor="1"
		module="CreateEstimate_ImageComparison">
		<detail>
			CreateEstimate_ImageComparison
		</detail>
		<varbindings>
		</varbindings>
		<item
			timeRelativeToTestSuiteStartTime="00:00.000"
			timeRelativeToTestModuleStartTime="00:53.175"
			timeWallClock="16:07:05"
			time="00:00.000"
			level="Error"
			category="Module">
			<message>
				The test run has been aborted by the user.
			</message>
			<metainfo
				loglvl="Error">
			</metainfo>
		</item>
	</activity>
</report>
